README.md

```diff
@@ -50,3 +50,60 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Add development build stage to dockerfile
+ docker build --target development --tag module5:dev .
+
+ ## Running the App in dev environment
+
+(base) RaviDommatas-Mac-mini:DevOps-Course-Starter rkdommata$ docker run -p 5000:5000 --env-file .env module5
+ * Serving Flask app "todo_app/app" (lazy loading)
+ * Environment: development
+ * Debug mode: on
+ * Running on http://0.0.0.0:5000/ (Press CTRL+C to quit)
+ * Restarting with stat
+ * Debugger is active!
+ * Debugger PIN: 307-202-561
+
+with mount bind on docker run :
+docker run -p 5000:5000 --mount type=bind,source="$(pwd)",target=/app --env-file .env module5
+ ## Running the App in PROD environment
+ ## added below entry to pyproject.toml
+ gunicorn = "20.0.4"
```

Comment:
Not sure what these two lines are doing here.

Also you can check the formatting with the preview button in the top right of VSCode or you can see it in GitHub https://github.com/domrk98git/DevOps-Course-Starter/blob/7b4d00ee61a84a78099bfa1f3820e7bc0da18f48/README.md